cps/shelf.py

```diff
@@ -403,7 +403,6 @@ def check_shelf_is_unique(title, is_public, shelf_id=False):
                                    .first() is None
 
         if not is_shelf_name_unique:
-            log.error("A public shelf with the name '{}' already exists.".format(title))
```

Comment:
Any change the line should be commented out and clarified, e.g. as follows?

```
# log.info("A public shelf with similar name '{}' already exists.".format(title))

Reply:
Ok

Reply:
Done